app-modules/ai/tests/Feature/Filament/Pages/ManageAiSettingsPageTest.php

```diff
@@ -66,7 +66,7 @@
 it('loads if you have the correct access to ai settings', function () {
     $user = User::factory()->licensed(LicenseType::ConversationalAi)->create();
 
-    $user->givePermissionTo(['product_admin.view-any']);
+    $user->givePermissionTo(['saas_global_admin.view-any']);
 
     actingAs($user);
```

Comment:
Since this is not controlled by a permission you can use `asSuperAdmin`

Reply:
@Orrison We won't be needing this test as we don't control it by permission ans asSuperAdmin() is already getting check in first test case.